tests/foreman/cli/test_repository.py

```diff
@@ -1865,16 +1865,78 @@ def test_positive_upload_content_srpm(self):
             local_file=get_data_file(SRPM_TO_UPLOAD),
             remote_file="/tmp/{0}".format(SRPM_TO_UPLOAD)
         )
+        # Upload SRPM
         result = Repository.upload_content({
             'name': new_repo['name'],
             'organization': new_repo['organization'],
             'path': "/tmp/{0}".format(SRPM_TO_UPLOAD),
             'product-id': new_repo['product']['id'],
+            'content-type': 'srpm',
         })
-        self.assertIn(
-            "Successfully uploaded file '{0}'".format(SRPM_TO_UPLOAD),
-            result[0]['message'],
+        assert "Successfully uploaded file '{0}'".format(SRPM_TO_UPLOAD) in result[0]['message']
+        assert int(Repository.info({'id': new_repo['id']})['content-counts']['source-rpms']) == 1
+
+        # Remove uploaded SRPM
+        Repository.remove_content({
+            'id': new_repo['id'],
+            'ids': [Srpm.list({'repository-id': new_repo['id']})[0]['id']],
+            'content-type': 'srpm'
+        })
+        assert int(Repository.info({'id': new_repo['id']})['content-counts']['source-rpms']) == 0
+
+    @tier2
+    def test_positive_srpm_list_end_to_end(self):
+        """Create repository,  upload, list and remove a SRPM content
```

Comment:
Create repository,  upload, list and remove an SRPM content

Reply:
same as above.